portfolio/src/main/webapp/script.js

```diff
@@ -53,8 +53,12 @@ function changeCat() {
 
 // Loads /data content from server and displays comments to the web page.
 function getComments() {
+  const languagesDropdown = document.getElementById("languages");
+  const language = languagesDropdown.options[languagesDropdown.selectedIndex].value;
```

Comment:
fyi: I think languagesDropdown has something like selectedValue or something like that that you can use directly.  No need to update, just letting you know, in case it does.